api-report/container-loader.api.md

```diff
@@ -152,6 +152,8 @@ export interface IFluidModuleWithDetails {
 
 // @public (undocumented)
 export interface ILoaderOptions extends ILoaderOptions_2 {
+    // (undocumented)
+    disableLocationRedirectionHandling?: boolean;
```

Comment:
we shouldn't add new options like this. use config instead

Reply:
The flow is - we try to open container and we fail. Storage returns error info with new URI.
Instead of patching all the layers with new URI, we allow exception to unroll fully stack, and start over with new URI.
Not sure I see how URI resolver can help here - it's hard to patch things like Container._resolvedUrl, it's much better to start over.